spec/requests/virus_scanning_spec.rb

```diff
@@ -56,7 +56,7 @@ def initialize
     get "/media/#{asset.id}/eicar.com"
     expect(response).to have_http_status(:not_found)
 
-    run_all_delayed_jobs
+    VirusScanWorker.drain
```

Comment:
Minor: I wonder whether this means we can stop automatically including the `DelayedJobHelpers` module and maybe even delete it? Maybe we can do that as a separate PR in which we remove all DJ related code...?

Reply:
I was thinking of removing all of the delayed job code in one PR - but yes, it does leave things a bit messy until then.